src/cli/scaffold/rust.rs

```diff
@@ -36,6 +37,20 @@ impl Scaffold for RustScaffold {
             vec!["init".to_owned(), "--vcs".to_owned(), "none".to_owned()],
         )?;
 
+        // add hdk-rust dependency by default
+        let cargo_file_path = base_path.as_ref().join(package::CARGO_FILE_NAME);
+
+        let mut f = OpenOptions::new()
+            .append(true)
+            .open(cargo_file_path)?;
+
+        // @TODO switch to crates.io ref when hdk-rust gets published
+        // @see https://github.com/holochain/holochain-cmd/issues/19
+        let hdk_dep: &str = "hdk = { git = \"https://github.com/holochain/hdk-rust\" }";
```

Comment:
Missing the 'branch' attribute

Reply:
It assumes master branch, and works in the case of hdk-rust, cause hdk-rust is on master. If that were changed, this would need to change

Reply:
I prefer explicit from implied but I'll just add my comment to my "I told you so" backlog :P

Reply:
For the benefit of avoiding said backlog, I will just add 'branch = "master"' then :p